code/modules/cargo/packs/engineering.dm

```diff
@@ -73,6 +73,17 @@
 	crate_name = "atmospherics hardsuit"
 	crate_type = /obj/structure/closet/crate/secure/engineering
 
+/datum/supply_pack/engineering/radhardsuit
+	name = "Radiation Hardsuit"
+	desc = "The Singulo is loose? Do you need to do a few changes to its containment and don't want to spent the rest of the shift under the shower? Get this Radiation Hardsuit! It protect from radiations and space only. Ask the CE to open."
+	cost = 5000
```

Comment:
This is not 5000"$" this is 10k in dollars. Could prob be 3500 do to being a passive hardsuit.

Reply:
You know crate prices better than me. Also I think it sell for the same price than the Engie hardsuit due to me not setting a price and it being a child of the hardsuit.